apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/repository/ItemRepository.java

```diff
@@ -35,6 +35,8 @@
 
   List<Item> findByNamespaceIdAndDataChangeLastModifiedTimeGreaterThan(Long namespaceId, Date date);
 
+  List<Item> findByKey(String key);
```

Comment:
I would suggest adding a pageable here in case we search out a huge amount of items.

Reply:
done.

Reply:
There are several pagables here, the pagable of apps, the pagable of namespaces and the pageable of items. Now we are using the same pageable and I think it may not bring the actual size of content we want. 
One way is to search out all the apps by items(by setting a large page size of items, e.g. 100) and do the pagination in memory, or we simply just return all the apps without pagination.

Reply:
I think the problem of different pageable is not a big problem. Because the trigger of getting next page is scrolling to bottom, and the view of content is small. So user dont make sense of different size of page. 
If the view is table , it's a big problem.

Reply:
okay, then we may need to add some comment in the code to mention we wrote the logic by purpose, in case someone is confused in the future :-)

btw, please solve the cla assistant issue